src/main/java/com/hedvig/claims/web/InternalController.kt

```diff
@@ -0,0 +1,595 @@
+                    claim.userId,
+                    claim.state,
+                    claim.reserve,
+                    claim.type,
+                    claim.audioURL,
+                    claim.registrationDate,
+                    claim.claimSource,
+                    claim.coveringEmployee
+                )
+            )
+        }
+        return ResponseEntity.ok(claims)
+    }
+
+    @GetMapping("/search")
+    fun search(req: ClaimsSearchRequestDTO): ClaimsSearchResultDTO {
+        return claimsQueryService.search(req)
+    }
+
+    @GetMapping("/listclaims/{userId}")
+    fun getClaimsByUserId(@PathVariable userId: String): List<ClaimDTO> {
+        return claimsRepository
+            .findByUserId(userId)
+            .map { claim: ClaimEntity ->
+                ClaimDTO(
+                    claim.id,
+                    claim.userId,
+                    claim.state,
+                    claim.reserve,
+                    claim.type,
+                    claim.audioURL,
+                    claim.registrationDate,
+                    claim.claimSource,
+                    claim.coveringEmployee
+                )
+            }
+    }
+
+    @GetMapping("/activeClaims/{userId}")
+    fun getActiveClaims(@PathVariable userId: String): ActiveClaimsDTO {
+        val activeClaims = claimsRepository.findByUserId(userId)
+            .count { c: ClaimEntity -> c.state == ClaimStates.OPEN }
+        return ActiveClaimsDTO(activeClaims)
+    }
+
+    @GetMapping("/stat")
+    fun claimsStatisticsByState(): Map<String, Long> {
+        return ClaimStates.values().associateBy(
+            { it.name },
+            claimsRepository::countByState
+        )
+    }
+
+    @GetMapping("/claim")
+    fun getClaim(@RequestParam claimID: String): ResponseEntity<ClaimDTO> {
+        val claim = claimsRepository.findById(claimID)
+            .orElseThrow { ResourceNotFoundException("Could not find claim with id:$claimID") }
+        return ResponseEntity.ok(ClaimDTO(claim))
+    }
+
+    @PostMapping("/adddataitem")
+    fun addDataItem(@RequestBody data: DataItemDTO): ResponseEntity<*> {
+        val uid = UUID.randomUUID()
+        val command = AddDataItemCommand(
+            uid.toString(),
+            data.claimID,
+            LocalDateTime.now(),
+            data.userId,
+            data.type,
+            data.name,
+            data.title,
+            data.received,
+            data.value
+        )
+        commandBus.sendAndWait<Any>(command)
+        return ResponseEntity.status(HttpStatus.NO_CONTENT).build<Any>()
+    }
+
+    @PostMapping("/addnote")
+    fun addNote(@RequestBody note: NoteDTO): ResponseEntity<*> {
+        val uid = UUID.randomUUID()
+        val command = AddNoteCommand(
+            uid.toString(),
+            note.claimID,
+            LocalDateTime.now(),
+            note.text,
+            note.userId,
+            note.fileURL
+        )
+        commandBus.sendAndWait<Any>(command)
```

Comment:
Since we don't return anything
```suggestion
        commandBus.sendAndWait<Void>(command)
```